libraries/testing/tester.cpp

```diff
@@ -1168,6 +1182,31 @@ namespace eosio { namespace testing {
 
    }
 
+   transaction_trace_ptr base_tester::set_finalizers(const vector<account_name>& finalizer_names) {
+      uint64_t fthreshold = finalizer_names.size() * 2 / 3;
```

Comment:
Should this be upper bounded in case `2/3` is not divisible?

Reply:
Doesn't really matter what this is set to. It is just a default value.